src/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualResponseBuilder.java

```diff
@@ -86,7 +86,15 @@ ResponseValues assembleResponse() throws TemplateModelException {
 		Individual dcoId = individual.getRelatedIndividual("http://info.deepcarbon.net/schema#hasDcoId");
 		if (dcoId != null) {	
 			body.put("dcoId", dcoId.getRdfsLabel());
+			System.out.println("world!!!");
 		}
+		// Trial for networkId
+		Individual networkId = individual.getRelatedIndividual("http://info.deepcarbon.net/schema#networkId");
```

Comment:
Is networkId an object property? I don't think it is. So not sure getObjectPropertyStatements would be the thing to use here.